### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by aborg0**
Attempt to fix the travis build:

- Native script fix (same as https://github.com/lihaoyi/fastparse/pull/180)
- `<?xml-` should be allowed, fixed it in `PITarget`
- xml entity references missed parenthesis
- `SymbolicKeywords` missed the lookahead for comments

Feel free to suggest changes.

### CODE REVIEW

**Comment by aborg0** [comment]
To be honest, I have no idea how `=/*?*/` can be parsed as an (infix operator type) `PlainId`: `P( !Keywords ~ Upper ~ IdRest(true) | VarId | Operator ~ (!OpChar | &("/*" | "//")) )`

I have tried to modify it to: `  val PlainId = P( !Keywords ~ ((Upper ~ IdRest(true)) | VarId | (Operator ~ (!OpChar | &("/*" | "//")))) )`, but no success. :(
`Keywords` look ok as there is a guard after the `=` symbolic keyword which should prevent combining `/` with `=`:

```
  val symbolKeywords = Seq(
    ":", ";", "=>", "=", "<-", "<:", "<%", ">:", "#", "@", "\u21d2", "\u2190"
  )

  val SymbolicKeywords = P{
    StringIn(symbolKeywords:_*) ~ !OpChar
  }
```

File: .travis.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ matrix:
     - env:
       - CI_SCRIPT: '"fastparseNative/test; fastparseByteNative/test; scalaparseNative/test; pythonparseNative/test; cssparseNative/test "'
       before_install:
-        - curl https://raw.githubusercontent.com/scala-native/scala-native/master/bin/travis_setup.sh | bash -x
+        - curl https://raw.githubusercontent.com/scala-native/scala-native/master/scripts/travis_setup.sh | bash -x
```
Comment (aborg0) [inline .travis.yml]: Same change as https://github.com/lihaoyi/fastparse/pull/180

**Comment by lihaoyi** [comment]
@aborg0  This looks good to me, if you're happy I'll merge it

**Comment by aborg0** [comment]
@lihaoyi Now I hope it is better, ready to get merged. I think these changes fix edge cases signaled by the test failures. If you prefer to merge #180 first, and there will be merge conflicts, I'll rebase.
